weave

```diff
@@ -1261,6 +1288,13 @@ case "$COMMAND" in
         ;;
     stop)
         [ $# -eq 0 ] || usage
+        stop $PROXY_CONTAINER_NAME "Proxy"
+        stop $DNS_CONTAINER_NAME "WeaveDNS"
+        stop $CONTAINER_NAME "Weave"
+        conntrack -D -p udp --dport $PORT >/dev/null 2>&1 || true
```

Comment:
I did consider pulling the body of `stop-router` out to a function to avoid duplicating this `conntrack` command, but it seemed unbalanced unless I did the same for `stop-proxy` and `stop-dns` too and that felt like overkill...